modules/op-about-finally_tasks.adoc

```diff
@@ -0,0 +1,64 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/understanding-openshift-pipelines.adoc
+
+[id="about-finally_tasks_{context}"]
+= Finally tasks
+
+The `finally` tasks are the final set of tasks specified using the `finally` field in the pipeline YAML file. A `finally` task always executes the tasks within the pipeline, irrespective of whether they are executed successfully. The `finally` tasks are executed in parallel after all the pipeline tasks are run, before the corresponding pipeline exits.
```

Comment:
> A `finally` task always executes the tasks within the pipeline, irrespective of whether they are executed successfully
what do `they` refer to?

Reply:
changing this entence to: A `finally` task always executes the tasks within the pipeline, irrespective of whether the pipeline runs are executed successfully.